sites/planet-bugzilla.groovy

```diff
@@ -8,9 +8,9 @@ node {
   }
 
   stage ('Build') {
-    docker.image('dhartnell/mozilla-planet-builder:3.0').inside('-u 0:0 -e https_proxy=$HTTPS_PROXY -e HTTPS_PROXY -e http_proxy=$HTTP_PROXY -e HTTP_PROXY') {
-      sh "/usr/local/bin/planet-prod.sh bugzilla"
-      sh "rsync -aq /data/genericrhel6/src/planet.bugzilla.org/* dst/"
+    docker.image('dhartnell/mozilla-planet-builder:4.7').inside("-u 0:0 -e https_proxy=$HTTPS_PROXY -e HTTPS_PROXY -e http_proxy=$HTTP_PROXY -e HTTP_PROXY -v /data/haul/.planet-cache/${env.JOB_NAME}:/data/efs/") {
```

Comment:
From 3.0 to 4.7 feels like a large step/change.

Any chance for a summary of what has changed at least?

GitHub comparaison link would also work just fine.

But generally, it’s hard to *review* a change and figure out the potential impact when we don’t see all that’s changedz

Reply:
This Groovy file is very old. Normally I would agree with you but the container isn't scoped to this one site. It is used daily for planet.mozilla.org and planet.mozilla.de. This only changes the actual site content that the Planet scripts generate. I don't have a good summary of what has changed since 3.0 - the container may not have even worked to successfully build site content at that point.

```
❯ docker run -it dhartnell/mozilla-planet-builder:4.7 bash
root@ebf6348746d3:/# /usr/local/bin/planet-build.sh bugzilla
root@ebf6348746d3:/# ls /data/genericrhel6/src/planet.bugzilla.org/
atom.xml  body_back.gif  foafroll.xml  index.html  opml.xml  planet.css  rss10.xml  rss20.xml
root@ebf6348746d3:/#
```